pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -137,8 +137,9 @@ export default defineComponent({
 
   data() {
     return {
-      updated:          false,
-      validationErrors: '',
+      updated:            false,
+      validationErrors:   '',
+      labelForIdentifier: `labeled-input-for-id-${ generateRandomAlphaString(12) }`
```

Comment:
nitpick: we could probably drop name this `inputId` to reduce how verbose the identifier is:

```suggestion
      inputId: `input-${ generateRandomAlphaString(12) }`
```